brayns/Brayns.cpp

```diff
@@ -193,6 +194,12 @@ struct Brayns::Impl : public PluginAPI
         auto& scene = _engine->getScene();
         auto& camera = _engine->getCamera();
 
+        if (_parametersManager.isAnyModified() || camera.isModified() ||
```

Comment:
why moved? the transferfunction.isModified() is not needed if you keep the markModified() in OSPRayScene, no?